compositor_pipeline/src/pipeline/whip_whep/validate_bearer_token.rs

```diff
@@ -0,0 +1,42 @@
+use std::time::Duration;
+
+use axum::http::HeaderValue;
+use rand::{rngs::StdRng, Rng, SeedableRng};
+use tokio::time::sleep;
+use tracing::warn;
+
+use crate::pipeline::whip_whep::error::WhipServerError;
+
```

Comment:
if you have separate file for token related stuff then you could rename this file and move token generation here too